spec/factories.rb

```diff
@@ -41,19 +41,29 @@
       with_icon { false }
       with_character { false }
       num_replies { 0 }
+
+      character { nil }
+      character_alias { nil }
+      icon { nil }
+      content { nil }
     end
     user
     board
-    description { "" }
-    content { "test content" }
+    description ""
```

Comment:
Aren't all values supposed to be in {}s? I thought that was a change we got bothered into by something?